homeassistant/components/apcupsd/sensor.py

```diff
@@ -34,392 +34,343 @@
         key="reg3",
         translation_key="register_3_fault",
-        name="UPS Register 3 Fault",
         entity_registry_enabled_default=False,
     ),
     "retpct": SensorEntityDescription(
         key="retpct",
         translation_key="restore_capacity",
-        name="UPS Restore Requirement",
         native_unit_of_measurement=PERCENTAGE,
     ),
     "selftest": SensorEntityDescription(
         key="selftest",
         translation_key="self_test_result",
-        name="UPS Self Test result",
     ),
     "sense": SensorEntityDescription(
         key="sense",
         translation_key="sensitivity",
-        name="UPS Sensitivity",
         entity_registry_enabled_default=False,
     ),
     "serialno": SensorEntityDescription(
         key="serialno",
         translation_key="serial_number",
-        name="UPS Serial Number",
         entity_registry_enabled_default=False,
     ),
     "starttime": SensorEntityDescription(
         key="starttime",
         translation_key="startup_time",
-        name="UPS Startup Time",
     ),
     "statflag": SensorEntityDescription(
         key="statflag",
         translation_key="online_status",
-        name="UPS Status Flag",
         entity_registry_enabled_default=False,
     ),
     "status": SensorEntityDescription(
         key="status",
         translation_key="status",
-        name="UPS Status",
     ),
     "stesti": SensorEntityDescription(
         key="stesti",
         translation_key="self_test_interval",
-        name="UPS Self Test Interval",
     ),
     "timeleft": SensorEntityDescription(
         key="timeleft",
         translation_key="time_left",
-        name="UPS Time Left",
         state_class=SensorStateClass.MEASUREMENT,
     ),
     "tonbatt": SensorEntityDescription(
         key="tonbatt",
         translation_key="time_on_battery",
-        name="UPS Time on Battery",
         state_class=SensorStateClass.TOTAL_INCREASING,
     ),
     "upsmode": SensorEntityDescription(
         key="upsmode",
         translation_key="ups_mode",
-        name="UPS Mode",
     ),
     "upsname": SensorEntityDescription(
         key="upsname",
         translation_key="ups_name",
-        name="UPS Name",
         entity_registry_enabled_default=False,
     ),
     "version": SensorEntityDescription(
         key="version",
         translation_key="version",
-        name="UPS Daemon Info",
         entity_registry_enabled_default=False,
     ),
     "xoffbat": SensorEntityDescription(
         key="xoffbat",
         translation_key="transfer_from_battery",
-        name="UPS Transfer from Battery",
     ),
     "xoffbatt": SensorEntityDescription(
         key="xoffbatt",
         translation_key="transfer_from_battery",
-        name="UPS Transfer from Battery",
     ),
     "xonbatt": SensorEntityDescription(
         key="xonbatt",
         translation_key="transfer_to_battery",
-        name="UPS Transfer to Battery",
     ),
```

Comment:
What does this return?

Reply:
["Date, time of last transfer to battery since apcupsd startup"](https://linux.die.net/man/8/apcaccess#:~:text=apcaccess%20is%20a%20program%20which,installed%2C%20properly%20configured%20and%20running). We should probably name this "Last transfer to battery date", thoughts?